src/api.js

```diff
@@ -0,0 +1,33 @@
+const apiBase = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/';
+
+export async function createGame(gameName) {
```

Comment:
* I suggest you use the ES6 `arrow` function in place of the `function` keyword this would make your code look more modern and similar to code bases out there in the wild so it's better for you to practice using ES6 syntax as early as possible 👍